e2e/_suites/fleet/services.go

```diff
@@ -387,7 +387,7 @@ func newDockerInstaller(ubi8 bool, version string) (ElasticAgentInstaller, error
 		WithArch(arch).
 		WithArtifact(artifact).
 		WithOS(os).
-		WithVersion(e2e.CheckPRVersion(version, agentVersionBase)) // sanitize version
```

Comment:
We will sanitize the version internally, see https://github.com/elastic/e2e-testing/pull/772/files#diff-d92db9d0c3d8b8a143e834b73bdfbcd816f191b556ae6439985409c475b58cd4R193-R194